tests/torchtune/modules/loss/test_liger_ce_loss.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+import torch.nn.functional as F
+from tests.test_utils import assert_expected, fixed_init_model, gpu_test
+from torch import nn
+from torch.distributed.tensor import DTensor
+from torch.optim import SGD
+from torchtune.modules.loss import LigerLinearCrossEntropy
+from torchtune.training.seed import set_seed
+
+
+@gpu_test(gpu_count=1)
```

Comment:
Let me edit and push some changes to these tests. I can run them myself then for you.